src/views/GovernanceTaskList/TaskListCta.tsx

```diff
@@ -206,8 +206,22 @@ export const DecisionCta = ({ intake }: { intake: SystemIntakeForm }) => {
     );
   }
 
+  if (intake.status === 'NOT_IT_REQUEST') {
+    return (
+      <span>
+        <b>Decision: </b>
```

Comment:
It'd be better to use `&nbsp;` rather than white space.

Reply:
good catch, fixed